fluent-resmgr/src/resource_manager.rs

```diff
@@ -138,3 +172,72 @@ impl BundleGenerator for ResourceManager {
         todo!()
     }
 }
+// coverage(on)
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use unic_langid::langid;
+
+    #[test]
+    fn caching() {
+        let res_mgr = ResourceManager::new("./tests/resources/{locale}/{res_id}".into());
+
+        let _bundle = res_mgr.get_bundle(vec![langid!("en-US")], vec!["test.ftl".into()]);
+        let res_1 = res_mgr.get_resource("test.ftl", "en-US");
+
+        let _bundle2 = res_mgr.get_bundle(vec![langid!("en-US")], vec!["test.ftl".into()]);
+        let res_2 = res_mgr.get_resource("test.ftl", "en-US");
+
+        assert!(
+            std::ptr::eq(res_1, res_2),
+            "The resources are cached in memory and reference the same thing."
+        );
+    }
+
+    // TODO - This API should return a Result instead.
+    // https://github.com/projectfluent/fluent-rs/issues/278
+    #[test]
+    #[should_panic]
+    fn get_resource_error() {
+        let res_mgr = ResourceManager::new("./tests/resources/{locale}/{res_id}".into());
+
+        let _bundle = res_mgr.get_bundle(vec![langid!("en-US")], vec!["test.ftl".into()]);
+        res_mgr.get_resource("nonexistent.ftl", "en-US");
+    }
+
+    // TODO - This API should return a Result instead.
+    // https://github.com/projectfluent/fluent-rs/issues/278
+    #[test]
+    #[should_panic]
+    fn get_bundle_error() {
+        let res_mgr = ResourceManager::new("./tests/resources/{locale}/{res_id}".into());
+        let _bundle = res_mgr.get_bundle(vec![langid!("en-US")], vec!["nonexistent.ftl".into()]);
+    }
+
+    // TODO - Syntax errors should be surfaced.
+    // https://github.com/projectfluent/fluent-rs/issues/280
+    #[test]
+    fn get_bundle_ignores_errors() {
+        // Note that this is the behavior at the time of the writing of this test.
```

Comment:
> thought

As a reader I'm not clear on what this comment is trying to express. 

"this is the behavior" doesn't mean very much to me. I wouldn't necessarily know if the behavior was updated and this comment was neglected to be removed.